gr-digital/include/gnuradio/digital/vblast_encoder_cc.h

```diff
@@ -0,0 +1,58 @@
+/* -*- c++ -*- */
+/* 
+ * Copyright 2018 Free Software Foundation, Inc.
+ * 
+ * This file is part of GNU Radio
+ * 
+ * GNU Radio is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3, or (at your option)
+ * any later version.
+ * 
+ * GNU Radio is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with GNU Radio; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+
+#ifndef INCLUDED_DIGITAL_VBLAST_ENCODER_CC_H
+#define INCLUDED_DIGITAL_VBLAST_ENCODER_CC_H
+
+#include <gnuradio/digital/api.h>
+#include <gnuradio/sync_decimator.h>
+
+namespace gr {
+  namespace digital {
+
+/*! \brief VBLAST encoder.
+ * Encodes a serial input data stream into a vertical transmission vector
+ * which is transmitted at once over multiple antennas.
+ * The functionality equals a multiplexing of the data.
```

Comment:
Isn't this what the GR block "stream to streams" does? I think we need to discuss with @marcusmueller if we need a second block doing the exact same thing just to have a nicely named block.

Reply:
I can see you point. Both options have their eligibility.